apollo-backend/src/main/resources/db/migration/V20180627150848__Add_concurrency_limit.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE environment ADD concurrency_limit int DEFAULT '0';
```

Comment:
ummm... so basically you're entering an invalid default value, no? 
If you're going for "flag to ignore this argument", I would suggest either null or -1 (0 seems like a potentially legit value, say, when you want to block certain deployments).
In that case, I'd also suggest having a "NO_CONCURRECY_LIMIT" \ "UNLIMITED_CONCURRENT_DEPLOYMENTS" constant in the KubernetesMonitor class and in isDeployedEnvironmentConcurrencyLimitPermitsDeployment checking that the value is different from that constant (and not just have it as part of the "check valid value" - it is clearer to the reader that way.

Reply:
`concurrencyLimit != null && concurrencyLimit > 0` is the check for this.
Any number >=0 make no sense in throttle, so added 0 as a default. 
I'll change to -1 to make it clearer, but 0 should never happen and there are blockers for that sake. Also added a `MINIMUM_CONCURRENCY_LIMIT = 1` so it will be clearer